default.nix

```diff
@@ -377,7 +377,7 @@ in let this = rec {
   };
 
   workOn = haskellPackages: package: (overrideCabal package (drv: {
-    buildToolDepends = (drv.buildToolDepends or []) ++ builtins.attrValues generalDevTools' {};
+    buildDepends = (drv.buildDepends or []) ++ builtins.attrValues (generalDevTools' {});
```

Comment:
This was changed the *other* direction in https://github.com/reflex-frp/reflex-platform/commit/b5c5c8b43d98e41534290bd7988a75d6e6a45045#diff-5712e736e0de6ba170577f8472c398e9L387

Why go back?

Reply:
Good catch, I think that was just a merge error.

Reply:
Looks like @ali-abrar explicitly changed this in recent commit